loops_lab.js

```diff
@@ -0,0 +1,193 @@
+// # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+//    let num = 60;
+// while (num >= 0) {
+//   console.log(num)
+//   num -= 1;
+// }
+//
+// let num = 60;
+// for (let num = 60; num >= 0; num -= 1) {
+//   console.log(num)
+// }
+
+
+// 2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+
+// let num = 0;
+// while (num <= 10) {
+// if (num % 2 === 0){
+//   console.log(num + ' is even')
+// } else {
+//   console.log(num + ' is odd')
+// }
+//   num++;
+// }
+//
+// for (let num = 0; num <= 10; num++) {
+//   if (num % 2 === 0){
+//     console.log(num + ' is even')
+//   }else {
+//     console.log( num + ' is odd')
+//   }
+// }
+
+
+// 3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
+// For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
+//   _Bonus_ think of another way to solve it.
+//     <details>
+//       <summary>
+//         Hint
+//       </summary>
+//       Find the final number and increment the loop by 9.
+//     </details>
+
+let num = 0;
+let num2 = 9;
+let num3 = num * 9;
+while (num <= 90){
```

Comment:
Maybe something like this will do the trick: 
let num = 0 
while(num <= 90) {
  if(num % 9 === 0) {
    console.log((num / 9) + " * 9 = " + num)
  }
  num++
}